index.js

```diff
@@ -68,7 +82,16 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies,id) {
+  let newObj = movies.find((movie)=>{
+    if(id === movie.imdbID){
+      return movie.title;        
+     }else if(!id){
+       return null;
+     }
+  })
+    return newObj;
+}
```

Comment:
```suggestion
function findById(movies,id) {
  let newObj = movies.find((movie)=>{
    if(id === movie.imdbID){
      return movie.title;        
  })
  if(newObj === undefined) {
    return null;
    } else {
    return newObj
    }
}
```
if the .find method can't find a match, it returns `undefined`. So `newObj` will be equal to `undefined` if a movie with the inputted id isn't found. But we want to return `null` if a movie with the inputted id isn't found, not `undefined`. So we just have to check if newObj is `undefined` and return `null` instead if that's the case.